pyproject.toml

```diff
@@ -76,9 +76,8 @@ dotnet = [
     "dotnetcore2==3.1.23; platform_system=='Linux'",
     "pywin32>=303; platform_system=='Windows'",
 ]
-
 doc = [
-    "ansys-sphinx-theme>=0.10.0,<1.1",
+    "ansys-sphinx-theme>=1.0.0,<1.3",
```

Comment:
Thanks for updating the range.

Reply:
That was on my TODO list since a few weeks :')